bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/WAToLNuA0123j_5f_LO_MLM/WAToLNuA0123j_5f_LO_MLM_proc_card.dat

```diff
@@ -0,0 +1,39 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm-ckm_no_b_mass
+
+# Define multiparticle labels
+define p = g u c d s u~ c~ d~ s~
```

Comment:
same here, you can remove the proton and jet definitions (which are wrong because they don't include the b quark and the model sm-ckm_no_b_mass is 5F...)

Reply:
Ok, thanks.